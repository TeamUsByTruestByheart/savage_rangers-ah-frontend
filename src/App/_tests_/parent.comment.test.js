import React from 'react';
import { shallow } from '../../enzyme';
import ParentComment from '../Components/Comment/parent.comment.body';

const parentWrapper = shallow(<ParentComment />);

describe('test the parent comment component to check if it renders appropriately', () => {
  test('should render the component with its props', () => {
    expect(parentWrapper.find('div').exists()).toBe(true);
    expect(parentWrapper).toMatchSnapshot();
  });

  test('toggle the hide and view reply', () => {
    parentWrapper.find('#view-replies').props().onClick();
    expect(parentWrapper.find('#view-replies').props().children[0]).toEqual('Hide Replies');
  });
});
