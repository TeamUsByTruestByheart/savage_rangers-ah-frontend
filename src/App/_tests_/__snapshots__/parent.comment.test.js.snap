// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the parent comment component to check if it renders appropriately should render the component with its props 1`] = `ShallowWrapper {}`;
